### DISCUSSION TITLE
Read time out

### DISCUSSION THREAD

**Discussion by ferdiko** [streamlink/streamlink#4142] [Q&A] [answered]
I'm downloading 10 live streams at the same time. However, for 3 of them I very frequently get a read timeout. They seem to be fine if I look at them on youtube (e.g. https://www.youtube.com/watch?v=H2wUyJiK5pg). What's the problem? This is my command:

`timeout 2h streamlink --hls-live-edge 99999 --hls-segment-threads 5 -o <FILENAME> <URL> best`

The timeout 2h is because I always want to save them in files of 2h.

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink/issues/4146
